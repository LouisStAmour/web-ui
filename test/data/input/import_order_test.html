<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="packages/web_ui/testing/testing.js"></script>
</head>
<body>
  <element name="foo-bar" extends="span">
    <template>hi</template>
  </element>
  <div>this should say "hello hi":{{topLevelVar}}<foo-bar></foo-bar></div>
  <script type="application/dart">
    library import_order_test;
    // this is a regression test for issue #389. Imports should always be before
    // 'part' directives.
    import 'dart:async';
    import 'dart:html';
    part 'import_order_part.dart';

    main() {
      topLevelVar = "hello";
      Timer.run(() => window.postMessage('done', '*'));
    }
  </script>
</body>
</html>

