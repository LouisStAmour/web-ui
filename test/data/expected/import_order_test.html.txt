Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x36
  RenderBlock {HTML} at (0,0) size 800x36
    RenderBody {BODY} at (8,8) size 784x20
      RenderBlock {DIV} at (0,0) size 784x20
        RenderText {#text} at (0,0) size 146x19
          text run at (0,0) width 146: "this should say \"hello hi\":"
        RenderText {#text} at (146,0) size 28x19
          text run at (146,0) width 28: "hello"
        RenderInline {FOO-BAR} at (0,0) size 10x19
          RenderText {#text} at (174,0) size 10x19
            text run at (174,0) width 10: "hi"
